Release/include/cpprest/details/fileio.h

```diff
@@ -54,16 +54,16 @@ struct _file_info
 
     // Positional data
 
-    size_t m_rdpos;
-    size_t m_wrpos;
+    utility::size64_t m_rdpos;
+    utility::size64_t m_wrpos;
     bool m_atend;
 
     // Input buffer
 
     size_t m_buffer_size; // The intended size of the buffer to read into.
     char* m_buffer;
 
-    size_t m_bufoff;                          // File position that the start of the buffer represents.
+    utility::size64_t m_bufoff;               // File position that the start of the buffer represents.
```

Comment:
This is wrong, and introduces several out of bounds reads by memcpy throughout the rest of the code